refapp/SKWebApi/appsettings.json

```diff
@@ -1,22 +1,35 @@
 {
-/*
-The following identity settings need to be configured
-before the project can be successfully executed.
-For more info see https://aka.ms/dotnet-template-ms-identity-platform
-*/
+  "CompletionConfig": {
+    "Label": "Completion",
+    "AIService": "AzureOpenAI",
+    "DeploymentOrModelId": "text-davinci-003",
+    "Endpoint": "https://YOUR-INSTANCE.openai.azure.com/",
+    "Key": "DONT_CHECK_IN_THIS_KEY"
```

Comment:
Add a comment on how to set this value using dotnet user-secrets. It's a *much* safer way to set these values in testing/hackathons. Maybe even comment the setting out. Same comment for all secrets config settings.

```
// dotnet user-secrets set "CompletionConfig:Key" "****"
// "Key": "**DO NOT CHECK IN**" 
```

Reply:
You may need to run `dotnet user-secrets init` next to the csproj from to add support for this.